conf/cassandra.yaml

```diff
@@ -1721,14 +1721,57 @@ client_encryption_options:
 # JMX SSL.
 # Similar to `client/server_encryption_options`, you can specify PEM-based
 # key material or customize the SSL configuration using `ssl_context_factory` in `jmx_encryption_options`.
-#jmx_encryption_options:
-#  enabled: true
-#  cipher_suites: [TLS_ECDHE_ECDSA_WITH_AES_256_GCM_SHA384,TLS_ECDHE_ECDSA_WITH_AES_128_GCM_SHA256]
-#  accepted_protocols: [TLSv1.2,TLSv1.3,TLSv1.1]
-#  keystore: conf/cassandra_ssl.keystore
-#  keystore_password: cassandra
-#  truststore: conf/cassandra_ssl.truststore
-#  truststore_password: cassandra
+#jmx_server_options:
+  # enabled: true
```

Comment:
@smiklosovic we also have `enabled` flag in encryption options below. Do we need both?

Reply:
Yes, we need both. You can have JMX started but not on SSL. These are two distinct properties. Disabling it here means no JMX server will be started at all.